oaknational/Oak-Web-Application
# fix: hack fix for blog post quote style

## Description

- puts a `<div>` round the quote author block, which seems to fix the issue :shrug: 
- **n.b. I couldn't find this issue elsewhere on the internet**
- i'm pretty certain it's a styled-components issue, since the props do **not** change, yet on refresh the class simply disappears from the `<p>`. I've [opened this issue](https://github.com/oaknational/Oak-Web-Application/issues/837) to track SC bugs we encounter.

_before refresh_
![image](https://user-images.githubusercontent.com/12934669/194049548-87152112-6ef0-4431-b3ac-a05a9e5eafc3.png)

_after refresh_
![image](https://user-images.githubusercontent.com/12934669/194049668-5e7c64b9-2d30-487e-9074-c78b567d7b18.png)


## Issue(s)

Fixes #739 

## How to test

1. Go to {cloud link}
3. You should see correect formatting of quotes even after refreshing the page

## Checklist
